tests/sentry/api/endpoints/test_organization_transaction_anomaly_detection.py

```diff
@@ -10,6 +10,7 @@
 from sentry.testutils import APITestCase, SnubaTestCase
 
 
+@freeze_time("2022-02-21")
```

Comment:
This will expire again at some point .. I suggest doing something like this https://github.com/getsentry/sentry/blob/f2f70bed7b2df8ef9209d1decc28b30f7d7d212d/tests/sentry/api/endpoints/test_organization_metric_data.py#L543

Reply:
I don't think so, the logic enforcing the limit is pinned by `freeze_time` (tested it with a date in 2016, works just as well).

Reply:
Will leave it to @nirajdewani to find a better long term solution if necessary.